app/assets/stylesheets/admin/questions/_questions.scss

```diff
@@ -78,3 +78,16 @@ section.questionnaires li.jqtree_common .jqtree-element
 .linked-document .help-block {
   margin: .25em 0;
 }
+
+.answer-wrapper.tree-view .decorated-numeric-input.form-element {
```

Comment:
@emitche do you have ideas about how to improve the maintainability of these styles? We want form-element on here because that's what simple_form uses to hide the input in read-only. But .answer-wrapper and .tree-view are only here to get enough specificity that these styles aren't overridden by simple_form styles.

Reply:
and/or is there a better place to put this new css?

Reply:
The placement and specificity is fine.

Reply:
thanks for taking a look!